modules/tinymce/src/core/main/ts/api/Settings.ts

```diff
@@ -129,6 +129,48 @@ const getPlaceholder = (editor: Editor): string =>
   // Fallback to the original elements placeholder if not set in the settings
   editor.getParam('placeholder', DOM.getAttrib(editor.getElement(), 'placeholder'), 'string');
 
+const getEventRoot = (editor: Editor): string => editor.getParam('event_root');
+
+const getServiceMessage = (editor: Editor): string => editor.getParam('service_message');
+
+const getTheme = (editor: Editor) => editor.getParam('theme');
+
+const shouldValidate = (editor: Editor): boolean => editor.getParam('validate');
+
+const isInlineBoundariesEnabled = (editor: Editor): boolean => editor.getParam('inline_boundaries') !== false;
+
+const getFormats = (editor: Editor) => editor.getParam('formats');
+
+const getPreviewStyles = (editor: Editor) => editor.getParam('preview_styles');
+
+const getCustomUiSelector = (editor: Editor): string => editor.getParam('custom_ui_selector', '', 'string');
+
+const getThemeUrl = (editor: Editor): string => editor.getParam('theme_url');
+
+const isInline = (editor: Editor): boolean => editor.getParam('inline');
+
+const hasHiddenInput = (editor: Editor): boolean => editor.getParam('hidden_input');
+
+const shouldPatchSubmit = (editor: Editor): boolean => editor.getParam('submit_patch');
+
+const isEncodingXml = (editor: Editor): boolean => editor.getParam('encoding') === 'xml';
+
+const shouldAddFormSubmitTrigger = (editor: Editor): boolean => editor.getParam('add_form_submit_trigger');
+
+const shouldAddUnloadTrigger = (editor: Editor): boolean => editor.getParam('add_unload_trigger');
+
+const hasForcedRootBlock = (editor: Editor): boolean => getForcedRootBlock(editor) !== '';
+
+const getCustomUndoRedoLevels = (editor: Editor): number => editor.getParam('custom_undo_redo_levels', 0, 'number');
+
+const shouldDisableNodeChange = (editor: Editor): boolean => editor.getParam('disable_nodechange');
+
+const isReadOnly = (editor: Editor): boolean => editor.getParam('readonly');
+
+const hasContentCssCors = (editor: Editor): boolean => getForcedRootBlock(editor) !== '';;
```

Comment:
Double semicolon at the end here.

Reply:
And this logic is definitely wrong, as the `getForcedRootBlock` function has nothing to do with the `content_css_cors` setting.

Reply:
I have no idea what happened there. Fixed.